.php_cs.dist

```diff
@@ -31,6 +31,12 @@ return \PhpCsFixer\Config::create()
         'phpdoc_align' => false,
         'phpdoc_summary' => false,
         'yoda_style' => false,
+        'ordered_class_elements' => [
+            'use_trait',
+            'constant_public',
+            'property_public',
```

Comment:
Does it mean that other settings from this rules are left by default? I'm interested in `'method_public', 'method_protected', 'method_private'` - exactly in this order

Reply:
Other settings will be as before: unordered default. 

That's 103 files changed if you want them in this order. And say goodbye to `git blame` along the way. I wouldn't recommend this.

Reply:
got it :(

Reply:
> That's 103 files changed if you want them in this order. And say goodbye to git blame along the way. I wouldn't recommend this.

If its something we really want, its better to just do it. The longer we wait, the bigger the impact of these things will be.

But if its not that important we can just skip it.